### PR TITLE
Troi integration

### PR DESCRIPTION
**Pull request by mayhem**
This PR adds the changes necessary to integrate the latest LB Radio. Improved error handling and changing how Troi was invoked, not much else.

The requirements.txt file will be updated before merging this PR, so please let me merge the PR after the corresponding troi PR is merged.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -22,7 +22,7 @@ pika == 1.2.1
 brainzutils@git+https://github.com/metabrainz/brainzutils-python.git@upgrade-deps
 spotipy>=2.22.1
 datasethoster@git+https://github.com/metabrainz/data-set-hoster.git@a1e12968af93d2f296a42a8094ebff83f3ed33f3
-troi@git+https://github.com/metabrainz/troi-recommendation-playground.git@v-2023-10-30.0
+troi@git+https://github.com/metabrainz/troi-recommendation-playground.git@integration-fixes
```
Comment (amCap1712) [inline requirements.txt:25]: Best to replace with exact commit hash to ensure latest commit is used.

**Review by amCap1712 — APPROVED** [review]
LGTM, Thanks!

File: requirements.txt (hunk 1 above)
Comment (mayhem) [reply requirements.txt:25]: Yes, once the troi PR is merged, I will make a release and then update this string, before merging.

**Comment by mayhem** [comment]
Test failures are unrelated.

**Pull request merged**